src/screens/PaymentDone/style.js

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from "react-native";
+import { theme1 } from "../../theme";
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: theme1.lightBrown,
+    flex: 1,
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  check: {
+    marginBottom: 10,
+  },
+  message: {
+    fontSize: 20,
+    fontWeight: "400",
+    color: theme1.gray,
+    textAlign: "center",
+    lineHeight: 25,
+  },
+  button: {
+    backgroundColor: theme1.gray,
+    padding: 15,
```

Comment:
Hi @umutzengi !

Reply:
Hi there @mustafadede if you want I can add the dimensions utility. So we can change static numbers in the style pages with responsive ones or if you add i can change it in my PR. I'm waiting your decision. Have a good day!